skstore/ts/src/skstore_api.ts

```diff
@@ -331,13 +308,13 @@ export interface TableHandle<R extends TJSON[]> {
    * Create a new eager reactive collection by mapping over each table entry
    * @returns {EHandle} An eager handle on the resulting collection
    */
-  map<
-    K extends TJSON,
-    V extends TJSON,
-    C extends new (...params: Param[]) => EntryMapper<R, K, V>,
-  >(
-    mapper: C,
-    ...params: EMParameters<K, V, R, C>
+  map<K extends TJSON, V extends TJSON>(
+    mapper: new (
+      eparams: EHandle<any, any>[],
+      lparams: LHandle<any, any>[],
+    ) => EntryMapper<R, K, V>,
+    eparams: EHandle<any, any>[],
+    lparams: LHandle<any, any>[],
```

Comment:
Could cut down boilerplate further by providing default (empty-array) arguments here

```suggestion
    eparams: EHandle<any, any>[] = [],
    lparams: LHandle<any, any>[] = [],
```

Reply:
I don't know JS well enough to know how it works when you have two optional args and you only want to pass one. I need to see if that is that common/easy.

Reply:
you can only leave off (positional) optional arguments after the last provided argument.

so if I had `foo(x=0,y=0,z=0) {...}` and wanted to provide `y`, I would have to provide `x` too but could still elide `z`.